src/compiler/compile/nodes/Element.ts

```diff
@@ -278,31 +279,32 @@ export default class Element extends Node {
 			// Errors
 
 			if (/(^[0-9-.])|[\^$@%&#?!|()[\]{}^*+~;]/.test(name)) {
-				component.error(attribute, {
+				return component.error(attribute, {
 					code: 'illegal-attribute',
 					message: `'${name}' is not a valid attribute name`
 				});
 			}
 
 			if (name === 'slot') {
 				if (!attribute.is_static) {
-					component.error(attribute, {
+					return component.error(attribute, {
 						code: 'invalid-slot-attribute',
 						message: 'slot attribute cannot have a dynamic value'
 					});
 				}
 
 				if (component.slot_outlets.has(name)) {
-					component.error(attribute, {
+					return component.error(attribute, {
 						code: 'duplicate-slot-attribute',
 						message: `Duplicate '${name}' slot`
 					});
 
-					component.slot_outlets.add(name);
+					// this code was unreachable. Still needed?
+					// component.slot_outlets.add(name);
```

Comment:
As indicated, this code was unreachable before this PR.

I think we could safely removed it since:
1. It is unreachable 😁
2. It does `set.add` in a `set.has` (what ?)